website/docs/api/inmemorylookupkb.mdx

```diff
@@ -194,26 +194,26 @@ to you.
 > candidates = kb.get_candidates((doc[0:2], doc[3:]))
 > ```
 
-| Name        | Description                                                                                  |
-| ----------- | -------------------------------------------------------------------------------------------- |
-| `mentions`  | The textual mention or alias. ~~Iterable[Span]~~                                             |
-| **RETURNS** | An iterable of iterable with relevant `Candidate` objects. ~~Iterable[Iterable[Candidate]]~~ |
+| Name        | Description                                                                                                  |
+| ----------- | ------------------------------------------------------------------------------------------------------------ |
+| `mentions`  | The textual mention or alias. ~~Iterable[Span]~~                                                             |
+| **RETURNS** | An iterable of iterable with relevant `InMemoryCandidate` objects. ~~Iterable[Iterable[InMemoryCandidate]]~~ |
 
 ## InMemoryLookupKB.get_alias_candidates {id="get_alias_candidates",tag="method"}
```

Comment:
Actually we should drop this, considering `_get_alias_candidates()` is private now. Alternatively we can re-expose the method.

Reply:
I agree that we can drop this, and have users query by mention instead of by alias as best practice.

Reply:
Done in https://github.com/explosion/spaCy/pull/12341/commits/3cfc1c6accb4f75b141b075aa53d8d9ae12166f0.